base/statistics.jl

```diff
@@ -493,6 +569,17 @@ end
 median!{T}(v::AbstractArray{T}) = median!(vec(v))
 median{T}(v::AbstractArray{T}) = median!(copy!(Array{T,1}(length(v)), v))
 
+"""
+    median(v[, region])
+
+Compute the median of whole array `v`, or optionally along the dimensions in `region`. For
+even number of elements no exact median element exists, so the result is equivalent to
+calculating mean of two median elements.
```

Comment:
some missing articles - "Compute the median of the whole array `v`, or optionally along the dimensions in `region`. For an even number of elements no exact median element exists, so the result is equivalent to calculating the mean of two median elements."

maybe a "the" before the "two" as well?